File: SiteVisitsNetCore2020Web/ClientApp/src/app/models/ExtractedTerm.ts
export class ExtractedTerm {
  //public id: number;
  public id: string;
  public visitId: string;
  public term: string;

  constructor(options: {
    //id: number,
    id: string,
    term: string
  }) {
    if (options) {
      this.id = options.id;
      this.term = options.term;
    }
  }

}

File: SiteVisitsNetCore2020Web/Models/ExtractedTerm.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace SiteVisitsNetCore2020Web.Models
{
    public class ExtractedTerm
    {
        //public int Id { get; set; }
        public Guid Id { get; set; }
        //[Key, ForeignKey("Visit")]
        [ForeignKey("Visit")]
        [Column(Order = 20)]
        public Guid VisitId { get; set; }
        public string Term { get; set; }
        public DateTime FirstOccurrence { get; set; }
        public DateTime DateCreated { get; set; }
    }
}
